snap/snapcraft.yaml

```diff
@@ -0,0 +1,21 @@
+name: foundationdb
+version: 6.0.18
+summary: A multi-model NoSQL database. 
+description:| 
+An open source, distributed, transactional key-value store.
+
+confinement: devmode
+base: core18
+
+architectures:
+  - build-on: amd64
+  
+parts:
+  foundationdb:
+    plugin: dump
+    source-type: tar
+    source: https://www.foundationdb.org/downloads/$SNAPCRAFT_PROJECT_VERSION/linux/fdb_$SNAPCRAFT_PROJECT_VERSION.tar.gz
+    
+apps:
+  foundationdb:
+   command: foundationdb 
```

Comment:
This still isn't a command that exists in the tarball you're downloading above?  Have you tested this snap?